src/Modules/UngradedModules.jl

```diff
@@ -1194,6 +1194,14 @@ function SubModuleOfFreeModule(F::FreeMod{L}, A::MatElem{L}) where {L}
   return subModule
 end
 
+function sparse_row(A::MatElem)
```

Comment:
Didn't we just remove this? Have you updated OSCAR master and then run `using Pkg ; Pkg.update()` to make sure you get the latest Hecke, Nemo etc.?

Reply:
Sorry, I had this open in a tab from last night and did not see the comments which already say this. But I can confirm: your code works fine for me with latest Oscar master and all packages up-to-date, except for the printing error.